### DISCUSSION TITLE
Small problems with Swedish translation

### DISCUSSION THREAD

**Discussion by PierreMesure** [open-webui/open-webui#3088] [General]
# Bug Report

## Description

**Bug Summary:**
Some Swedish translations cause issues or are just subpar.
For instance, the date format was converted to [something](https://github.com/open-webui/open-webui/blob/0917fa6f4ac015810211d85eb41da5d335762952/src/lib/i18n/locales/sv-SE/translation.json#L305) that doesn't parse the date correctly.

![image](https://github.com/open-webui/open-webui/assets/7223028/99f4f52a-163d-4bec-b2c3-737539e3ca4f)

## Environment
- **Open WebUI Version:** 0.3.3

## Note

I can go through all the Swedish translations this week and submit a PR.

**Comment by PierreMesure** [comment]
There is also a small UI issue with this panel, it disappears with zoom at 150%.
![image](https://github.com/open-webui/open-webui/assets/7223028/e0677821-42f3-4e0c-9c08-9e4f19b8dc07)

**Comment by tjbck** [comment]
PR Welcome!
